executor/admin.go

```diff
@@ -433,7 +432,7 @@ func (e *RecoverIndexExec) backfillIndexInTxn(ctx context.Context, txn kv.Transa
 		}
 
 		recordKey := e.table.RecordKey(row.handle)
-		err := txn.LockKeys(ctx, nil, 0, kv.LockAlwaysWait, time.Now(), recordKey)
+		err := txn.LockKeys(ctx, new(kv.LockCtx), recordKey)
```

Comment:
The default value of time.Time is '0001-01-01 00:00:00 +0000 UTC'. Is it OK?

Reply:
It's only used when forUpdateTS > 0, so it's ok.